webapi/src/main/scala/org/knora/webapi/slice/admin/api/service/RestPermissionService.scala

```diff
@@ -0,0 +1,84 @@
+package org.knora.webapi.slice.admin.api.service
```

Comment:
license header

Reply:
Thanks for pointing it out, fixed in 419902b9ff3767e436540c7bfdb256cabc2b55e4